keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -168,11 +169,14 @@ Ensure existing tests (for preempting PriorityClasses) do not break.
 
 #### Beta (v1.19):
 
-- [ ] Add integration test for NonPreemptingPriority.
-- [ ] Graduate NonPreemptingPriority to Beta.
-- [ ] Update documents to reflect the changes.
-
+- [x] Add integration test for NonPreemptingPriority.
+- [x] Graduate NonPreemptingPriority to Beta.
+- [x] Update documents to reflect the changes.
 
+#### Stable (v1.24):
+- [x] No negative feedback.
+- [ ] Graduate NonPreemptingPriority to GA.
+- [ ] Update documents to reflect the changes.
```

Comment:
you can remove the checkboxes (along with the beta ones) to indicate they're complete; otherwise, you have to revisit to check them.

Reply:
Done